components/platform/platform.c

```diff
@@ -354,6 +497,19 @@ void platform_uart_stop( unsigned id )
 int platform_uart_get_config(unsigned id, uint32_t *baudp, uint32_t *databitsp, uint32_t *parityp, uint32_t *stopbitsp) {
     int err;
 
+#if CONFIG_ESP_CONSOLE_USB_SERIAL_JTAG
+    if (id == 0) {
+      // Return dummy values
```

Comment:
Would it not be more honest to just return an error code here instead of dummy values?

Reply:
I was nervous that it might frighten some code that thinks that it ought to be able to get the config....

Reply:
Good code should be able to handle jump scares! :D

Come to think of it though, isn't the USB device accepting serial configuration settings? And shouldn't we return those (even if they aren't being used)?

Reply:
It accepts settings from the PC side, but there isn't any way to get those from the ESP32 side (as far as I can tell). I *think* that the CDC device is hardware and just exposes a few registers to look like a UART. In particular, on the C3 I know that you cannot reprogram the USB to like anything except a CDC device. I don't think that is true for all variants of the ESP32.